### ISSUE TITLE
Unable to resolve dependency tree with React 17 and NPM 7

### ISSUE THREAD

**Issue by wiesys** [tinacms/tinacms#1809]
## Description

`@tinacms/react-forms@0.40.0` has a dependency `react-beautiful-dnd@"^11.0.5"`, which has a peer dependency of  
`react@"^16.8.5"`.

So if I'm using React 17 in my project, is it not possible to install Tina without using "--force" :)?

### Steps to reproduce

```
npm i react tinacms
```

### Expected result

That the installation would went fine.

### Actual result

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"*" from the root project
npm ERR!   peer react@">=16.8" from tinacms@0.40.0
npm ERR!   node_modules/tinacms
npm ERR!     tinacms@"*" from the root project
npm ERR!   16 more (react-dom, styled-components, @tinacms/fields, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.5" from react-beautiful-dnd@11.0.5
npm ERR! node_modules/@tinacms/react-forms/node_modules/react-beautiful-dnd
npm ERR!   react-beautiful-dnd@"^11.0.5" from @tinacms/react-forms@0.40.0
npm ERR!   node_modules/@tinacms/react-forms
npm ERR!     @tinacms/react-forms@"^0.40.0" from tinacms@0.40.0
npm ERR!     node_modules/tinacms
npm ERR!       tinacms@"*" from the root project
npm ERR!     2 more (@tinacms/react-sidebar, @tinacms/react-toolbar)
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
```

### Environment

NPV v7.7.6

#### Tinacms Package Versions

I wasn't able to install Tina, but the package causing problems is in my opinion is: `@tinacms/react-forms@0.40.0`, which needs to update it's peer dependency `react-beautiful-dnd@"^11.0.5"`.

**Comment by wiesys** [comment]
I managed to make it work with `npm i --legacy-peer-deps react tinacms`, but still updating `react-beautiful-dnd` would be nice :)

**Comment by DirtyF** [comment]
drag-n-drop has been dropped for now, I wonder if this package is still needed. 🤔

**Comment by arpi0003** [comment]
This outdated version of `react-beautiful-dnd` is causing other issues for me as well. I'm getting the same error referenced by the original post in this thread https://community.tinacms.org/t/group-list-not-working-in-the-useform-nextjs-project/354/6. I added a resolution to my package.json file to use version 13.1.0 of `react-beautiful-dnd` and everything seemed to work, but it would be nice if it was updated in Tina's dependencies.
